src/controllers/staffControllers.ts

```diff
@@ -0,0 +1,99 @@
+import { ObjectId } from "mongodb";
+import Staff from "../models/Staff";
+
+import { type Request, type Response } from "express";
+
+const createStaff = async (req: Request, res: Response) => {
+    try {
+        const { firstName, lastName, phoneNumber, email, dateJoined } =
+            req.body;
+
+        if (!(firstName && lastName && phoneNumber && email && dateJoined)) {
+            return res.status(400).send({ message: "Missing Required Field" });
+        }
+
+        const databaseId = await Staff.find()
+            .sort({ id: -1 })
+            .limit(1)
+            .then((docs: any) => {
+                return docs[0].get("id");
+            })
+            .catch((err: any) => {
+                console.log(err);
+            });
```

Comment:
I see what you're doing here, but this is a bit too complicated than what we need. I know this code was in a previous change++ project, but in that project, each document needed a unique, 6 number id for identification purposes. We don't need that for staff, so this can be simplified greatly! (Look at createUser here: https://github.com/ChangePlusPlusVandy/hfc-backend/blob/main/controllers/users.controllers.js)